components/00-base/functions/map.scss

```diff
@@ -0,0 +1,12 @@
+@use 'sass:map';
+
+/// Map deep get
```

Comment:
Good call adding the deep-get function. This is a lot easier to read/use compared to using nested `map-get`'s